index.html

```diff
@@ -585,15 +585,15 @@ <h3>Use Cases and Requirements</h3>
 
       <section id="conformance" class="normative">
         <p>
-A <dfn>conforming document</dfn> is any concrete expression of the data model
-that complies with the normative statements in this specification.
-Specifically, all relevant normative statements in Sections
+A <dfn>conforming document</dfn> is a compact JSON-LD document that
+complies with all of the relevant "MUST" statements in this specification.
+Specifically, the relevant normative "MUST" statements in Sections
 <a href="#basic-concepts"></a>, <a href="#advanced-concepts"></a>, and
-<a href="#syntaxes"></a> of this document MUST be enforced. A serialization
-format for the conforming document MUST be deterministic, bi-directional,
-and lossless as described in Section <a href="#syntaxes"></a>.
-The <a>conforming document</a> MAY be transmitted or stored in any such
-serialization format.
+<a href="#syntaxes"></a> of this document MUST be enforced. Other serialization
+formats for the conforming document MAY be used, and if they are, further
+guidance in Section <a href="#ecosystem-compatibility"></a> MUST be
+followed. The <a>conforming document</a> MAY be transmitted
+or stored in any such serialization format.
```

Comment:
I'm not sure what this means for the result type of the verification algorithm. Does that need to be a compact JSON-LD document, or can it be one of the other serialization formats?

Reply:
The WG didn't like that language, as of the last meeting. It's been removed in a515e6fbfed5b28bd31af56fac790836567b8242.